app/src/main/java/ch/sdp/vibester/database/DataGetter.kt

```diff
@@ -104,7 +104,10 @@ class DataGetter @Inject constructor() {
      * @param uid id of the new user
      */
     fun createUser(email: String, username: String, callback: (String) -> Unit, uid: String) {
-        var newUser = User(username, "", email, 0, 0, 0, 0, uid)
+        val newUser = User()
```

Comment:
Is this worth testing?

Reply:
I believe no, since it is the database call and not testable